Feature Request(CLI): Using custom `.svelte-kit` locations

### Describe the bug

I am using non-default setup for sveltekit where files are not in .svelte-kit. When I run `pnpm dlx shadcn-svelte@next add ...` the command fails at `pnpm dlx svelte-kit sync` with `ERR_PNPM_DLX_NO_BIN  No binaries found in svelte-kit`.

This is my svelte.config.js file:
```
const config = {
  kit: {
    ...
    files: {
      hooks: {
        universal: process.env.CLIENT ? 'src/client/hooks' : 'src/admin/hooks',
        server: process.env.CLIENT ? 'src/client/hooks.server' : 'src/admin/hooks.server',
        client: process.env.CLIENT ? 'src/client/hooks.client' : 'src/admin/hooks.client'
      },
      routes: process.env.CLIENT ? 'src/client/routes' : 'src/admin/routes'
    },
    outDir: `.dist.${process.env.CLIENT ? 'client' : 'admin'}`
  },
  ...
}
```

NOTE: I am using the `next` version of shadcn

### Reproduction

1. Create a new SvelteKit project
2. Set `files` property to non-default values in `svelte.config.js`
3. Run shadcn cli to set it up with the project

### Logs

```bash
ERR_PNPM_DLX_NO_BIN  No binaries found in svelte-kit
```


### System Info

```bash
MacOS
```


### Severity

annoyance

@ieedan not sure which step you are reffering to. I've ran pnpm install multiple times. When I get to the `pnpm dlx  shadcn-svelte@next init` this is the error I am getting:

```
┌   shadcn-svelte  v1.0.0-next.2 
│
└  Error: Command failed with exit code 1: pnpm dlx svelte-kit sync
┌   shadcn-svelte  v1.0.0-next.2 
│
└  Error: Command failed with exit code 1: pnpm dlx svelte-kit sync
 ERR_PNPM_DLX_NO_BIN  No binaries found in svelte-kit
    at makeError (file:///Users/ronen/Library/Caches/pnpm/dlx/g4vh4unkxgiirjz23rsfeewzdm/1932ca44204-22f5/node_modules/.pnpm/execa@7.2.0/node_modules/execa/lib/error.js:60:11)
    at handlePromise (file:///Users/ronen/Library/Caches/pnpm/dlx/g4vh4unkxgiirjz23rsfeewzdm/1932ca44204-22f5/node_modules/.pnpm/execa@7.2.0/node_modules/execa/index.js:124:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async sn (file:///Users/ronen/Library/Caches/pnpm/dlx/g4vh4unkxgiirjz23rsfeewzdm/1932ca44204-22f5/node_modules/.pnpm/shadcn-svelte@1.0.0-next.2/node_modules/shadcn-svelte/dist/index.js:59:2258)
    at async et (file:///Users/ronen/Library/Caches/pnpm/dlx/g4vh4unkxgiirjz23rsfeewzdm/1932ca44204-22f5/node_modules/.pnpm/shadcn-svelte@1.0.0-next.2/node_modules/shadcn-svelte/dist/index.js:59:3551)
    at async Te (file:///Users/ronen/Library/Caches/pnpm/dlx/g4vh4unkxgiirjz23rsfeewzdm/1932ca44204-22f5/node_modules/.pnpm/shadcn-svelte@1.0.0-next.2/node_modules/shadcn-svelte/dist/index.js:59:3509)
    at async Command.<anonymous> (file:///Users/ronen/Library/Caches/pnpm/dlx/g4vh4unkxgiirjz23rsfeewzdm/1932ca44204-22f5/node_modules/.pnpm/shadcn-svelte@1.0.0-next.2/node_modules/shadcn-svelte/dist/index.js:181:1163)
```

--- a/.changeset/ten-spiders-do.md
+++ b/.changeset/ten-spiders-do.md
@@ -2,4 +2,4 @@
 "shadcn-svelte": patch
 ---
 
-ensure sync executes local binary
+fix: Ensure `svelte-kit sync` executes locally

--- a/.changeset/ten-spiders-do.md
+++ b/.changeset/ten-spiders-do.md
@@ -1,0 +1,5 @@
+---
+"shadcn-svelte": patch
+---
+
+ensure sync executes local binary

--- a/packages/cli/src/utils/sveltekit.ts
+++ b/packages/cli/src/utils/sveltekit.ts
@@ -15,7 +15,7 @@ export async function syncSvelteKit(cwd: string) {
 
 		agent ??= { agent: "npm", name: "npm" };
 
-		const cmd = resolveCommand(agent.agent, "execute", ["svelte-kit", "sync"]);
+		const cmd = resolveCommand(agent.agent, "execute-local", ["svelte-kit", "sync"]);
 		if (cmd) {
 			await execa(cmd.command, cmd.args, {
 				cwd,

